### DISCUSSION TITLE
Make test results show the average WPM

### DISCUSSION THREAD

**Discussion by Doob123123** [monkeytypegame/monkeytype#3765] [Ideas] [answered]
Rather than the test results showing the last recorded WPM score, make it so that it shows the average for the test. It seems something like this being implemented could be more accurate.
![Screenshot 2022-11-22 2 48 51 PM](https://user-images.githubusercontent.com/80001887/203407791-ca88c8d3-c313-4275-b768-1c8f0f4ecc85.png)
As shown in the last set of numbers (being hovered over on the end of the line graph) It shows just about the same result WPM as it does the last WPM. If it already shows the average, then the accuracy needs to be improved. There was a test that I completed that said the score was 112 WPM, but the average of the test was 119.

**Comment by Miodec** [comment, marked as answer]
Its already the average. It wouldn't make sense to show the last wpm point (for example if you stopped at the end of the test your wpm would be 0)
